mix.exs

```diff
@@ -11,13 +11,18 @@ defmodule Membrane.RTP.H264.MixProject do
       elixir: "~> 1.12",
       elixirc_paths: elixirc_paths(Mix.env()),
       start_permanent: Mix.env() == :prod,
```

Comment:
In `docs()` function there is `formatter` field. Are you sure that there can be `HTML`? In documentation there is `html`.